docs/api.rst

```diff
@@ -5,7 +5,7 @@ API
     :noindex:
     :synopsis: public Jinja API
 
-This document describes the API to Jinja and not the template language.  It
+This document describes the API to Jinja and not the template language (for that, see [here](https://jinja.palletsprojects.com/en/2.11.x/templates/).  It
```

Comment:
The docs are rst, not markdown. I don't think your syntax works there.

Reply:
Ah... Let me fix that